### ISSUE TITLE
I18n for the Wrangler

### ISSUE THREAD

**Issue by nathanaelhoun** [gabrieljackson/mattermost-plugin-wrangler#97]
First, I want to thank you for this plugin, the work-in-progress UI is great and the features are really useful. And they are especially with new-comers, who haven't really understood the functioning of Mattermost WRT threads and channels.
And so, as these new comers aren't often comfortable with English when it's not their primary language, it could be great to have the "A thread has been moved" and "Someone wrangled one of your messages into a thread for you" messages translated for the user.

**Comment by gabrieljackson** [comment]
I totally agree and will work on getting I18n support into the plugin. Thanks for the suggestion!
